docs/dxp/7.x/en/installation-and-upgrades/upgrading-liferay-dxp/upgrade-basics/upgrading-to-a-new-liferay-portal-ce-rolling-release.md

```diff
@@ -1,32 +1,16 @@
 # Upgrading to a New Liferay Portal CE Rolling Release
 
-Fixes for Liferay Portal CE are delivered *rolling releases*. That is fixes for a Portal CE GA are published in all newer GA releases for that version of Poral CE. For example, fixes to CE 7.3 GA1 bugs are included in all newer CE 7.3 GA releases (i.e., GA2+). Here are the steps for upgrading a Portal CE installation to a new rolling release:
+Fixes and improvements for Liferay Portal CE are delivered *rolling releases*. That is fixes for a Portal CE GA are published in all newer GA releases for that version of Poral CE. For example, fixes to CE 7.3 GA1 bugs are included in all newer CE 7.3 GA releases (i.e., GA2+).
 
-1. Shut down your Liferay Portal CE server.
-
-1. Delete cache.
-
-    Delete the `[Liferay Home]/osgi/state` folder.
-
-    ```bash
-    rm -rf [Liferay Home]/osgi/state
-    ```
-
-    Empty the `[Liferay Home]/work` folder.
-
-    ```bash
-    rm -rf work/*
-    ```
+Here are the steps for upgrading a Portal CE installation to a new rolling release (this only applies when upgrading between GAs in the same Liferay CE version):
 
-    Delete the application server cache. Please consult the application vendor documentation on where where to find the cache.
+1. Download the new Portal CE GA release files.
 
-    ```note::
-       If a module's changes are only internal, the changes are invisible to the OSGi framework, the module stays installed, and the module's state persists. Clearing the OSGi bundle state information before the next DXP startup ensures that such modules reinstall with the appropriate state.
-    ```
+1. Shut down your Liferay Portal CE server.
 
 1. [Back up your DXP system](../../maintaining-a-liferay-dxp-installation/backing-up.md), including any files that you have added or edited.
 
-1. Install the new Portal CE GA release files on top of your existing installation.
+1. Install the new Portal CE GA release files in the previous installation folder on in a new folder.
```

Comment:
@jhinkey maybe here we can say unzip or maybe we can remove the first step and say here download and unzip, that would be better.